posthog/api/test/test_annotation.py

```diff
@@ -120,29 +120,6 @@ def test_updating_annotation(self, mock_capture):
             self.user, "annotation updated", {"scope": "organization", "date_marker": None},
         )
 
-    def test_deleting_annotation_of_other_team_prevented(self):
```

Comment:
Not sure why this test is deleted? Is it not still a valid test to ensure the cross-team deletion isn't possible?

Reply:
I thought about keeping it for the signal it gives about expected behaviour but now no deletion is possible it felt redundant to assert no deletion is possible cross team